.github/workflows/continious-integration.yaml

```diff
@@ -0,0 +1,145 @@
+name: "Continuous Integration"
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**/*.md"
+      - "LICENSE"
+  pull_request:
+    paths-ignore:
+      - "**/*.md"
+      - "LICENSE"
+
+jobs:
+  link:
+    name: Lint
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+
+      - name: Install Composer dependencies
+        run: composer update --prefer-dist --no-interaction --no-ansi
+
+      - name: "Run coding style checking"
+        run: composer lint
+
+  phpunit:
+    name: Test PHP ${{ matrix.php-version }} & Symfony ${{ matrix.symfony-version }} ${{ matrix.composer-flags }}
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - php-version: '7.4'
+            symfony-version: '4.4.*'
+            composer-flags: '--prefer-lowest'
+          - php-version: '7.4'
+            symfony-version: '5.4.*'
+            composer-flags: ''
+          - php-version: '8.0'
+            symfony-version: '6.0.*'
+            composer-flags: ''
+          - php-version: '8.1'
+            symfony-version: '6.0.*'
+            composer-flags: ''
+          - php-version: '8.1'
+            symfony-version: '6.1.*'
+            composer-flags: ''
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+
+      # We need Flex to enfore Symfony version with `SYMFONY_REQUIRE` env var
+      - name: Install Symfony Flex
+        run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
+
+      - name: Enable Symfony Flex
+        run: composer global config --no-plugins allow-plugins.symfony/flex true
```

Comment:
I'd merge this step with the previous one, something like `Install and enable Symfony Flex`